gcore/gdal.h

```diff
@@ -2513,18 +2517,27 @@ void CPL_DLL GDALAttributeFreeRawResult(GDALAttributeH hAttr, GByte *raw,
                                         size_t nSize);
 const char CPL_DLL *GDALAttributeReadAsString(GDALAttributeH hAttr);
 int CPL_DLL GDALAttributeReadAsInt(GDALAttributeH hAttr);
+GInt64 CPL_DLL GDALAttributeReadAsLong(GDALAttributeH hAttr);
```

Comment:
Please avoid the "Long" name as it is never clear how large that type is. Use "Int64" instead. Applies to all new additions. You can also use int64_t . GInt64 is legacy at that point